app/views/users/index.html.erb

```diff
@@ -1,15 +1,16 @@
 <h1>Users Index</h1>
-<ul>
-  <li>
-    <p>User 1</p>
-    <p># of Posts: x</p>
-  </li>
-  <li>
-    <p>User 2</p>
-    <p># of Posts: x</p>
-  </li>
-  <li>
-    <p>User 3</p>
-    <p># of Posts: x</p>
-  </li>
-</ul>
+<% @users.each do |user| %>
+  <div class="user-line">
+    <%= link_to image_tag(user.photo, class: 'user-photo'), user_path(user) %>
+    <div class="name-counter">
+      <span class="name">
+        <%= link_to user.name, user_path(user) %>
+      </span>
+      <div>
+        <span class="right-top"></span>
+        <span class="counter">Number of posts: <%= user.posts_counter %></span>
+        <span class="right"></span>
+      </div>
+    </div>
+  </div>
```

Comment:
👍 Excellent progress on your project! However, to enhance the codebase, consider the following suggestion:

- [ ] Redundancy Alert! There's repetitive code for user details in `user#index` , `user#show`, and `post#index` erb file. In line with the DRY (Don't Repeat Yourself) principle, it's advisable to eliminate redundancy and maintain a clean codebase. One effective solution is to create [partials](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials). Partials are reusable templates that can streamline rendering processes and reduce code duplication. Please update your code and keep your code DRY as much as possible. 🙏 

By implementing partials, you'll improve maintainability and adhere to best practices. Keep up the stellar work! 🚀👩‍💻